libs/types/src/schema.test.ts

```diff
@@ -800,7 +800,7 @@ test('safeParseElectionDefinition computes the election hash', () => {
   expect(
     t.safeParseElectionDefinition(electionData).unsafeUnwrap().electionHash
   ).toMatchInlineSnapshot(
-    `"d5366378eeccc2fd38953e6e34c3069dea0dca4b7a8f5c789f3d108dc1807d3c"`
+    `"89d5b1e06c888634c318593eda29331adb9a74011178cdd7c9555cca06838e7d"`
```

Comment:
In some places where we're actually computing the election hash, we should expect it to change and include that in our testing.